src/Spectre.libGenetic.Tests/GenerationTest.cpp

```diff
@@ -27,6 +27,33 @@ namespace
 using namespace spectre::algorithm::genetic;
 using namespace spectre::core::exception;
 
+const auto seed = 0ul;
+
+TEST(GenerationInitialization, initializes)
```

Comment:
We were talking about moving this functionality (random initialization) from `Generation` to a kind of factory.

Reply:
done